diff --git a/node_modules/@storybook/addon-knobs/dist/index.d.ts b/node_modules/@storybook/addon-knobs/dist/index.d.ts
index edb0dfb..6b0ee48 100644
--- a/node_modules/@storybook/addon-knobs/dist/index.d.ts
+++ b/node_modules/@storybook/addon-knobs/dist/index.d.ts
@@ -13,5 +13,5 @@ export declare function date(name: string, value?: Date, groupId?: string): numb
 export declare function button(name: string, callback: ButtonTypeOnClickProp, groupId?: string): undefined;
 export declare function files(name: string, accept: string, value?: string[], groupId?: string): string[];
 export declare function optionsKnob<T extends OptionsTypeKnobSingleValue>(name: string, valuesObj: OptionsTypeOptionsProp<T>, value: OptionsTypeKnobValue<T>, optionsObj: OptionsKnobOptions, groupId?: string): T;
-export declare const withKnobs: import("@storybook/preview-api/dist/storybook-channel-mock-a19cd233").M;
+export declare const withKnobs: any;
 export * from './shared';
